Modules/core/BRTransaction.h

```diff
@@ -109,9 +109,9 @@ typedef struct {
 //    char name[31]; // MAX 31 Bytes
     size_t nameLen;
     uint64_t amount;     // 8 Bytes
-    int64_t unit;        // 1 Byte
-    int64_t reissuable;  // 1 Byte
-    int64_t hasIPFS;     // 1 Byte
+    int unit;        // 1 Byte
```

Comment:
```suggestion
    int8_t unit;        // 1 Byte
```

Reply:
this is an old commit, I changed those to int:
    int unit;      
    int reissuable;
    int hasIPFS;     
for swift to C interfacing but I still manage to add them as varints

Reply:
They shouldn't be int either.  They should be specified by number of bits.